### PR TITLE
commit

### PR DESCRIPTION
**Pull request by aibrahem550**
add checkin data export

### CODE REVIEW

**Comment by aibrahem550** [comment]
@dmdesimone take a look at these changes please.

File: configuration/reports/reportdescriptors/dataexports/sql/checkin.sql [hunk 1]
```diff
@@ -0,0 +1,77 @@
+SELECT encounter_type_id INTO @enctype FROM encounter_type et WHERE uuid='55a0d3ea-a4d7-4e88-8f01-5aceb2d3c61b';
+
+DROP TABLE IF EXISTS checkin_details;
```
Comment (dmdesimone) [inline configuration/reports/reportdescriptors/dataexports/sql/checkin.sql:3]: make this a temporary table so it gets cleaned up

**Review by dmdesimone — CHANGES_REQUESTED** [review]
didn't review the SQL in detail but requested the one change about the temp table.  Otherwise looks good.

File: configuration/reports/reportdescriptors/dataexports/sql/checkin.sql (hunk 1 above)
Comment (aibrahem550) [reply configuration/reports/reportdescriptors/dataexports/sql/checkin.sql:3]: I see its already temporary 
image

**Pull request merged**